smac/intensification/intensification.py

```diff
@@ -188,6 +194,10 @@ def intensify(self, challengers, incumbent, run_history, aggregate_func,
 
                 # Line 10
                 missing_runs = list(inc_inst_seeds - chall_inst_seeds)
+                
+                if (not first_run) and missing_runs:
+                    first_run = True
+                    chall_indx += 1
```

Comment:
Sorry, I don't get it. 
If `to_run` is not empty, we will have new runs.
`to_run` is a truncated list of `missing_runs`. I don't see a case in which `missing_runs` is not empty but `to_run` is empty since `N` is at least 1.
So, if `missing_runs` is not empty, we will have at least one run of the challenger.

However, there could an issue with adaptive capping. If we picked the challenger for the second time, adaptive capping could prevent us to have further runs on this challenger.

Therefore, I moved the code further down.